### PR TITLE
Finished chp 12 NB - except last 2 exercises

### PR DESCRIPTION
**Pull request by AlexAndorra**
Should be ready to merge - will solve the last two exercises when Ordered Logistic sampling issue is resolved

### CODE REVIEW

File: Rethinking/Data/bangladesh.csv [hunk 1]
```diff
@@ -0,0 +1,3869 @@
+"woman";"district";"use.contraception";"living.children";"age.centered";"urban"
+
+1;1;0;4;18.44;1
+
```
Comment (aloctavodia) [inline Rethinking/Data/bangladesh.csv]: please remove blank lines :-)

Comment (AlexAndorra) [reply Rethinking/Data/bangladesh.csv]: This again?? I swear I don't know where they come from :laughing: Will do ASAP :wink:

**Comment by aloctavodia** [comment]
thanks @AlexAndorra!

**Comment by AlexAndorra** [comment]
My pleasure ;)

**Pull request merged**